enlighten/measurement/MeasurementFactory.py

```diff
@@ -7,19 +7,19 @@
 import os
 import re
 
-from .ColumnFileParser import ColumnFileParser
-from .ExportFileParser import ExportFileParser
-from .TextFileParser   import TextFileParser
-from .SPCFileParser    import SPCFileParser
-from .ThumbnailWidget  import ThumbnailWidget
-from .DashFileParser   import DashFileParser
-from .Measurement      import Measurement
+from enlighten.parser.ColumnFileParser import ColumnFileParser
+from enlighten.parser.ExportFileParser import ExportFileParser
+from enlighten.parser.TextFileParser import TextFileParser
+from enlighten.parser.SPCFileParser import SPCFileParser
+from enlighten.ui.ThumbnailWidget import ThumbnailWidget
+from enlighten.parser.DashFileParser import DashFileParser
+from enlighten.measurement import Measurement
```

Comment:
Pretty sure this should be:
`from enlighten.measurement.Measurement import Measurement`